src/components/UTIL_JobProgressLightning.component

```diff
@@ -45,10 +45,10 @@
     <apex:outputPanel layout="block" id="jobProgressContainer" styleClass="slds">
```

Comment:
Alignment appears to be off on the progress component.
![screen shot 2018-09-10 at 9 28 53 am](https://user-images.githubusercontent.com/450473/45303717-0e44e100-b4dc-11e8-99ac-d37eb1ad42bc.png)

Reply:
Good catch. Fixed.